src/display/api.js

```diff
@@ -143,6 +152,16 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
  *   reading built-in CMap files. Providing a custom factory is useful for
  *   environments without Fetch API or `XMLHttpRequest` support, such as
  *   Node.js. The default value is {DOMCMapReaderFactory}.
+ * @property {string} [standardFontDataUrl] - The URL where the standard font
+ *   files are located. Include the trailing slash.
+ * @property {boolean} [standardFontDataWorkerFetch] - Enable fetching the font
+ *   data from the worker thread. When `true`, StandardFontDataFactory will be
+ *   ignored. The default value is `true` in web environment and `false` for
```

Comment:
Let's remove the ```When `true`, StandardFontDataFactory will be ignored.``` sentence here, since we should always fallback to a main-thread fetch if all else fails.